src/app/beer_garden/api/http/handlers/v1/garden.py

```diff
@@ -68,7 +78,9 @@ async def delete(self, garden_name):
         """
         garden = self.get_or_raise(Garden, GARDEN_DELETE, name=garden_name)
 
-        await self.client(Operation(operation_type="GARDEN_DELETE", args=[garden.name]))
+        await self.client(
+            BrewtilsOperation(operation_type="GARDEN_DELETE", args=[garden.name])
```

Comment:
`Operation` becomes `BrewtilsOperation` for consistency with we've been doing elsewhere.

Reply:
I like having the *Brewtils* prefix if for no other reason than I can later know exactly where to kill it with fire when the time comes ;)